general_considerations.md

```diff
@@ -221,17 +302,32 @@ sign/authorize this UHPO transaction.
 
 We don't ever want to actually have to broadcast this UHPO set transaction
 except in the case of pool failure. Similar to other optimistic protocols like
-Lightning, we will withhold this transaction from bitcoin and update it
-out-of-band with respect to bitcoin. With each new block we will update the UHPO
-set transaction to account for any new shares since the last block mined by the
-pool.
+Lightning, the DMP will withhold this transaction from bitcoin and update it
+out-of-band with respect to bitcoin. With each new block the UHPO set is updated
+to account for any new shares since the last block mined by the pool.
+
 
 Furthermore a decentralized mining pool should support "withdrawal" by hashers.
 This would take the form of a special message or transaction sent to the pool
 (and agreed by consensus within the pool) to *remove* a hasher's output from the
 UHPO set transaction, and create a new separate transaction which pays that
 hasher, [authorizes](#payout-authorization) it, and broadcasts it to bitcoin.
 
+[kptodo]
+
+Address the question: This introduces another consensus protocol, and if an attacker can
+attack this protocol, it is game over again.
+
+[/kptodo]
```

Comment:
The withdrawal request is a transaction in the braid, and is mined into beads, so it uses the same consensus as the braid itself and does not use a separate consensus mechanism.